src/ComServerHelpers/BaseActivationFactory.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Runtime.Versioning;
+
+namespace ComServerHelpers;
```

Comment:
We should really think about what actual namespace names to use. And type names.